### PR TITLE
feat(cactus-connector-fabric): add get transaction receipt by tx id

### PR DESCRIPTION
**Pull request by RafaelAPB**
Authored-by: Eduardo Vasques eduardovasques10@tecnico.ulisboa.pt
Co-authored-by: Rafael Belchior rafael.belchior@tecnico.ulisboa.pt
Signed-off-by: Rafael Belchior rafael.belchior@tecnico.ulisboa.pt

**Pull Request Requirements**
- [x] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [x] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [x] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [x] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [x] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

File: packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts [hunk 1]
```diff
@@ -158,10 +159,17 @@ import {
 } from "./common/utils";
 import { findAndReplaceFabricLoggingSpec } from "./common/find-and-replace-fabric-logging-spec";
 import { deployContractGoSourceImplFabricV256 } from "./deploy-contract-go-source/deploy-contract-go-source-impl-fabric-v2-5-6";
+import { Observable, ReplaySubject } from "rxjs";
 
 const { loadFromConfig } = require("fabric-network/lib/impl/ccp/networkconfig");
 assertFabricFunctionIsAvailable(loadFromConfig, "loadFromConfig");
 
+export interface RunTxReqWithTxId {
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts:167]: @RafaelAPB I recommend using the `I` prewfix when naming interfaces in general. Not too important but definitely recommended to keep it as a convention.

**Review by petermetz — CHANGES_REQUESTED** [review]
@RafaelAPB Please fix the DCO, otherwise looking good to me.

**Review by petermetz — APPROVED** [review]
@RafaelAPB I'm not sure what was wrong with the DCO but it's passing now so I'm OK.

Specifically to what you wrote: It's important to note that the GPG signing keys don't come into play at all from the DCO perspective.
The DCO is just a plain string comparison of the email in your commit message (the "signed-off by" line on the bottom) and the email that you have set up on your github account. So the email you have associated with your GPG key does not get examined at all.

**Comment by petermetz** [comment]
Oops, there's also a merge conflict, please let me know if you need help with that

**Comment by petermetz** [comment]
@hyperledger-cacti/cacti-maintainers bump

**Review by sandeepnRES — APPROVED** [review]

**Comment by petermetz** [comment]
@RafaelAPB The codegen task was failing but only due to a flake. I'm re-running it now, hopefully we can merge it after this execution.

**Pull request merged**